src/components/MainMenu/index.js

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import '../../App.css'; 
 import { NavLink } from 'react-router-dom';
 import { Navbar, Nav } from 'react-bootstrap';
+import nav_logo from '../../assets/nav_logo.svg';
+import './index.css';
 
 const MainMenu = (props) => (
           <Navbar bg="dark" variant="dark" fixed="top">
-                <Navbar.Brand href="/home">Logo</Navbar.Brand>
+                <Navbar.Brand as={NavLink} to="/"><img className="nav-logo" src={nav_logo} alt="" /></Navbar.Brand>
```

Comment:
Please move <img> tag to the new line and add some text in the alt attribute like: "Starwars Logo"